### PR TITLE
DN4J-11: add helm chart for NEO4J v5

### PR DESCRIPTION
**Pull request by enys**

### CODE REVIEW

File: charts/neo4jv5-internal/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Neo4J LKE chart
```
Comment (enys) [inline charts/neo4jv5-internal/README.md:1]: Not updated yet

File: charts/neo4jv5-internal/values.preprod.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+          value: 'true'
+        volumeMounts:
+          - name: data
+            mountPath: /tmp/git/
+      - name: neo4j-restore
+          # command: ['ls', '-lah', '/data/git-sync/', '&&', '/data/git-sync/docker-neo4j/scripts/dataset-download.sh']
+        command: ['/data/docker-neo4j/scripts/dataset-download.sh', '-p', '/data']
+        envFrom:
+          - secretRef:
+              name: neo4j-dev-nexus-dataset # imports NEXUS_TOKEN
```
Comment (enys) [inline charts/neo4jv5-internal/values.preprod.yaml:47]: revert back to preprod

**Review by thibveni — APPROVED** [review]

File: charts/neo4jv5-internal/values.preprod.yaml (hunk 2 above)
Comment (thibveni) [reply charts/neo4jv5-internal/values.preprod.yaml:46]: just seeing this, what do you mean ?

Comment (enys) [reply charts/neo4jv5-internal/values.preprod.yaml:46]: it's a note for myself, in neo4j-preprod namespace env the secret is neoj4-preprod ...
and we can't template this.

**Pull request merged**